--- a/net/ipv4/tcp_ipv4.c
+++ b/net/ipv4/tcp_ipv4.c
@@ -43,9 +43,9 @@
  	net->ipv4.sysctl_tcp_min_rtt_wlen = 300;
  	net->ipv4.sysctl_tcp_autocorking = 1;
  	net->ipv4.sysctl_tcp_invalid_ratelimit = HZ/2;
-	net->ipv4.sysctl_tcp_pacing_ss_ratio = 200;
-	net->ipv4.sysctl_tcp_pacing_ca_ratio = 120;
+	net->ipv4.sysctl_tcp_pacing_ss_ratio = 150;
+	net->ipv4.sysctl_tcp_pacing_ca_ratio = 110;
  	if (net != &init_net) {
  		memcpy(net->ipv4.sysctl_tcp_rmem,
  		       init_net.ipv4.sysctl_tcp_rmem,
+		       sizeof(init_net.ipv4.sysctl_tcp_rmem));

